crates/bevy_input/src/gamepad.rs

```diff
@@ -398,7 +407,8 @@ impl GamepadAxis {
 ///
 /// The [`GamepadSettings`] are used inside of the [`gamepad_event_system`], but are never written to
 /// inside of `bevy`. To modify these settings, mutate the corresponding resource.
-#[derive(Resource, Default, Debug)]
+#[derive(Resource, Default, Debug, Reflect, FromReflect)]
+#[reflect(Debug)]
```

Comment:
You could also register the `ReflectDefault` type data.

Reply:
I'll also add `ReflectSerialize` and `ReflectDeserialize` behind a `cfg_attr` where applicable. Thanks!